src/shared/containers/ResourcePlugins.tsx

```diff
@@ -96,8 +94,8 @@ const ResourcePlugins: React.FunctionComponent<{
                 style={{ marginBottom: 10 }}
                 key={`builtin-plugin-${plugin}`}
               >
-                <ErrorBoundary fallback={Fallback}>
-                  {builtInComponent}
+                <ErrorBoundary key={plugin} fallback={Fallback}>
+                  <>{builtInComponent}</>
```

Comment:
we don't need a key in `ErrorBoundary` (the key only needed for the top level wrapper under map), neither the fragment for `builtInComponent`